py3status/modules/do_not_disturb.py

```diff
@@ -102,6 +103,20 @@ def toggle(self, state):
                 self.parent.py3.command_run("pkill -SIGTERM dunst")
 
 
+class Mako(Notification):
+    """
+    Mako Notification.
+    """
+
+    def toggle(self, state):
+        if state:
+            # pause
+            self.parent.py3.command_run("makoctl set invisible=1")
```

Comment:
(Alternative) Dunno if it's too long. Here's a line or two.
```python
self.parent.py3.command_run("makoctl set invisible={}".format(int(bool(state))))
```

Reply:
Neat, I keep this one.